### ISSUE TITLE
Decreased myocardial glycogen content

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11357]
**Preferred term label:**

**Synonyms**
Myocardial glycogen depletion
Glycogen depletion in cardiac muscle

**Definition (free text, please give PubMed ID)**
PMID: 40536305 Glycogen storage disorders are a group of genetic disorders affecting glucose homeostasis in the body. Muscular glycogen stores are essential for liberating glucose for energy supply during bursts of activity and sustained muscle work. Muscle glycogen storage disease 0 (GSD0B) is associated with biallelic variants in GYS1 causing muscular glycogen synthetase deficiency and has previously been identified as a cause of sudden cardiac arrest in childhood. Muscle biopsies of affected living relatives demonstrated a paucity of glycogen staining.
Patient 11:  **No glycogen in cardiomyocytes**. Normal liver glycogen
Patient 14: **Depletion of glycogen Heart structurally normal, normal size and wall thicknes**

PMID: 17928598 The cardiac-muscle specimen  showed myocyte hypertrophy with enlarged nuclei but no myocyte disarray or fibrosis. **No glycogen was detected in cardiomyocytes by PAS stainling**

PMID: 21958591 Muscle glycogen storage disease 0 (GSD0) is caused by **glycogen depletion in skeletal and cardiac muscles** due to deficiency of
glycogen synthase 1 (GYS1), which is encoded by the GYS1 gene.

**Parent term (use hpo.jax.org/app)**

Abnormal muscle glycogen content HP:0012269

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:137625 Glycogen storage disease due to muscle and heart glycogen synthase deficiency

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001440
- label: Decreased myocardial glycogen content
- parents: [Abnormal myocardium morphology (HP:0001637)](https://hpo.jax.org/app/browse/term/HP:0001637)
- pmid: PMID:40536305|PMID:17928598|PMID:21958591
- def: The amount of glycogen in the myocardial tissue of the heart is below the lower limit of normal.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11357
- synonyms: Glycogen depletion in cardiac muscle tissue(NONE)
